design/api.go

```diff
@@ -30,4 +31,13 @@ var _ = API("alm", func() {
 		Header("Authorization")
 	})
 
+	ResponseTemplate(Created, func(pattern string) {
+		Description("Resource created")
+		Status(201)
+		Headers(func() {
+			Header("Location", String, "href to created resource", func() {
+				Pattern(pattern)
```

Comment:
Currently, it's not used much by the Goa generated code itself but rendered in the swagger definitions.

The future plan for Goa is to use it to e.g. simplify the Context response object to have methods like:

```go
ctx.Response().Location("xxx")
```